books/projects/x86isa/machine/dispatch-macros.lisp

```diff
@@ -318,9 +320,11 @@
                       (eql proc-mode #.*compatibility-mode*)
                       (eql proc-mode #.*protected-mode*)))))
     :ud)
-   ((eq type-id :type-vex-gpr)
+   ((or (eq type-id :type-vex-gpr)
+        ;; BOZO Sol -- it seems like maybe this is a synonym?
```

Comment:
I'm not familiar with this part of the code, so I don't think I can provide an informed opinion. I can't even find the pre-existing `:type-vex-gpr` anywhere in the code other than here (perhaps it occurs in generated code?).